src/components/calendar/CalendarPage.jsx

```diff
@@ -70,11 +102,13 @@ const styles = theme => ({
     justifyContent: 'space-between',
   },
   button: {
-    border: 'none',
+    // border: 'none',
```

Comment:
It's safe to remove this line entirely!

Reply:
Okieee I'll remove other comments as well :)